src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -0,0 +1,118 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+/** Servlet that returns all account related info (links to login pages/account info). */
+@WebServlet("/account")
+public class AccountData extends HttpServlet {
+
+  public final class JsonHelper{
+        private final boolean loggedIn;
+        private final String logInOutURL;
+        private final String signUpURL;
+        public JsonHelper(Boolean loggedIn, String logInOutURL, String signUpURL){
+            this.loggedIn=loggedIn;
+            this.logInOutURL=logInOutURL;
+            this.signUpURL = signUpURL;
+          }
+        public boolean getLoggedIn(){
```

Comment:
Some inconsistent styling with the space around operations and before the parentheses.